code/obj/machinery/plantpot.dm

```diff
@@ -1187,10 +1189,15 @@
 					seedcount++
 				getamount--
 
+			var/harvest_string = "You harvest [cropcount] item"
+			if (cropcount > 1)
+				harvest_string += "s"
 			if(seedcount)
-				boutput(user, "<span class='notice'>You harvested [cropcount] items and [seedcount] seeds.</span>")
-			else
-				boutput(user, "<span class='notice'>You harvested [cropcount] items.</span>")
+				harvest_string += " and [seedcount] seed"
+				if (seedcount > 1)
+					harvest_string += "s"
+			harvest_string += ".</span>"
```

Comment:
Could we get this done without use of `+=` for string concatenation? It's decidedly bad, performance-wise. An alternative would look something like (untested):
```
			var/list/harvest_string = list("You harvest [cropcount] item")
			if (cropcount > 1)
				harvest_string += "s"
			if (seedcount)
				harvest_string += " and [seedcount] seed"
				if (seedcount > 1)
					harvest_string += "s"
			boutput(user, "<span style=\"color:blue\">[harvest_string.Join()]</span>")
```
Basically, we're changing repeated string concatenations with a list of things we build up then stick together at the end.

Reply:
re: stick together, `string_list.Join()`

Reply:
I can definitely look into fixing this now!

Reply:
Resolved in latest commit, and replaced color:blue with class='notice' for consistency